pages.fr/common/git-add.md

```diff
@@ -0,0 +1,24 @@
+# git add
+
+> Ajoute les fichiers modifiés à l'index.
+> Plus d'informations : <https://git-scm.com/docs/git-add>.
+
+- Ajouter un ficher à l'index :
+
+`git add {{path/to/file}}`
```

Comment:
```suggestion
`git add {{chemin/vers/fichier}}`
```
You seem to have left all the user tokens in English. Please also translate the tokens (I'm writing this only once but it applies to *all* user tokens in all the pages. You can take a look at existing French pages for examples.